templates/index.template.html

```diff
@@ -4,10 +4,10 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>UI component pattern library</title>
-  <link rel="stylesheet" href="/styles/pattern-library.css">
+  <link rel="stylesheet" href="/{{ css_bundle }}">
```

Comment:
I suggest the whole path should be a variable, including the leading `/`. If the file was moved for example, that part might need to change.